lib/order.rb

```diff
@@ -0,0 +1,90 @@
+#Products structure { "banana" => 1.99, "cracker" => 3.00 }
+#Fulfuillment_status: :pending, :paid, :processing, :shipped, or :complete
+require_relative 'customer.rb'
+
+class Order
+attr_accessor :products, :customer, :fulfillment_status
```

Comment:
Unless you have a specific need to alter these during runtime outside the class itself, I'd probably make all of these just `_reader`s.